src/Type/InterfaceType/NodeWithRevisions.php

```diff
@@ -26,12 +26,12 @@ public static function register_type( TypeRegistry $type_registry ) {
 						'description' => __( 'If the current node is a revision, this field exposes the node this is a revision of. Returns null if the node is not a revision of another node.', 'wp-graphql' ),
 						'resolve'     => function( Post $post, $args, AppContext $context, ResolveInfo $info ) {
 							// @codingStandardsIgnoreLine.
-							if ( 'revision' !== $post->post_type || ! isset( $post->parentId ) || ! absint( $post->parentId ) ) {
+							if ( 'revision' !== $post->post_type || ! isset( $post->parentDatabaseId ) || ! absint( $post->parentDatabaseId ) ) {
 								return null;
 							}
 
 							// @codingStandardsIgnoreLine.
-							return DataSource::resolve_post_object( $post->parentId, $context );
+							return DataSource::resolve_post_object( $post->parentDatabaseId, $context );
```

Comment:
FWIW, you could get around adding this field to the model by doing `$post->data->parent` here. Not saying that's the right way forward here, just throwing it out as a possible solution.

Reply:
@CodeProKid we don't load the whole object for the Parent in the Model though, eh? You're suggesting that we do load it in the Model?

Reply:
@jasonbahl nope, that'll return the parent DB ID though.